.github/workflows/main.yml

```diff
@@ -0,0 +1,64 @@
+name: vscode-bitbake CI/CD
+
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [20]
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Change Directory
+      run: cd path/to/your/project
```

Comment:
This template is not adapted yet to our project. I suggest squashing it directly with the next commit.

Reply:
done